staging/src/k8s.io/apiserver/pkg/audit/policy/checker.go

```diff
@@ -36,9 +36,26 @@ type Checker interface {
 
 // NewChecker creates a new policy checker.
 func NewChecker(policy *audit.Policy) Checker {
+	for i, rule := range policy.Rules {
+		policy.Rules[i].OmitStages = unionStages(policy.OmitStages, rule.OmitStages)
```

Comment:
nit: prefer:
```
policy.Rules[i].OmitStages = sets.NewString(policy.OmitStages, rule.OmitStages).UnsortedList()
```

Reply:
This will get such error:
```
cannot use rule.OmitStages (type []"k8s.io/kubernetes/vendor/k8s.io/apiserver/pkg/apis/audit".Stage) as type []string in argument to sets.NewString
```

I don't want to convert them to string and convert back, so I wrote this util function.

Reply:
Ugh, yeah, makes sense. Not the first time I've run into this...